linebot/api.py

```diff
@@ -1001,4 +1001,5 @@ def __check_error(response):
             pass
         else:
             error = Error.new_from_json_dict(response.json)
-            raise LineBotApiError(response.status_code, error)
+            request_id = response.headers.get('X-Line-Request-Id')
+            raise LineBotApiError(response.status_code, request_id, error)
```

Comment:
[imo]
How about adding keywords(request_id= and error=) to clarify both are keyword parameters?